testgrid/config.yaml

```diff
@@ -6863,22 +6886,42 @@ dashboards:
     test_group_name: istio-circleci-e2e-mixer-noauth-v1alpha3-v2
     alert_options:
       alert_mail_to_addresses: istio-oncall@googlegroups.com
+  - name: circleci-e2e-mixer-noauth-v1alpha3-v2-mcp
+    test_group_name: istio-circleci-e2e-mixer-noauth-v1alpha3-v2-mcp
+    alert_options:
+      alert_mail_to_addresses: istio-oncall@googlegroups.com
   - name: circleci-e2e-simple
     test_group_name: istio-circleci-e2e-simple
     alert_options:
       alert_mail_to_addresses: istio-oncall@googlegroups.com
+  - name: circleci-e2e-simple-mcp
+    test_group_name: istio-circleci-e2e-simple-mcp
+    alert_options:
+      alert_mail_to_addresses: istio-oncall@googlegroups.com
   - name: circleci-e2e-dashboard
     test_group_name: istio-circleci-e2e-dashboard
     alert_options:
       alert_mail_to_addresses: istio-oncall@googlegroups.com
+  - name: circleci-e2e-dashboard-mcp
+    test_group_name: istio-circleci-e2e-dashboard-mcp
+    alert_options:
+      alert_mail_to_addresses: istio-oncall@googlegroups.com
   - name: circleci-e2e-pilot-auth-v1alpha3-v2
     test_group_name: istio-circleci-e2e-pilot-auth-v1alpha3-v2
     alert_options:
       alert_mail_to_addresses: istio-oncall@googlegroups.com
+  - name: circleci-e2e-pilot-auth-v1alpha3-v2-mcp
+    test_group_name: istio-circleci-e2e-pilot-auth-v1alpha3-v2-mcp
+    alert_options:
+      alert_mail_to_addresses: istio-oncall@googlegroups.com
   - name: circleci-e2e-pilot-noauth-v1alpha3-v2
     test_group_name: istio-circleci-e2e-pilot-noauth-v1alpha3-v2
     alert_options:
       alert_mail_to_addresses: istio-oncall@googlegroups.com
+  - name: circleci-e2e-pilot-noauth-v1alpha3-v2-mcp
```

Comment:
Perhaps you want to group all mcp tests together? Foe now only you folks care about them and they will be folded in the other tests once the results are stable. It will be easier for you guys to see, and less noisy to other people who don't need to care about these for now.